rastervision_pytorch_learner/rastervision/pytorch_learner/learner.py

```diff
@@ -239,13 +248,14 @@ def main(self):
                 self.train()
                 if cfg.save_model_bundle:
                     self.save_model_bundle()
+        else:
+            self.load_checkpoint()
 
-        self.load_checkpoint()
+        self.stop_tensorboard()
+        self.sync_to_cloud()
```

Comment:
Shouldn't `sync_to_cloud()` should be called after `eval_model('valid')`?

Reply:
Yep, thanks for catching that. I believe it was only the `stop_tensorboard()` call I wanted to move up.